app/src/main/res/values/themes.xml

```diff
@@ -1,7 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-
-    <style name="Theme.Animite" parent="android:Theme.Material.Light.NoActionBar">
-        <item name="android:statusBarColor">@color/purple_700</item>
-    </style>
+    <style name="Theme.Animite" parent="android:Theme.Material.Light.NoActionBar"/>
```

Comment:
You might consider removing Theme.Animite entirely and just specifying the parent directly in your manifest